packages/evolution-backend/src/services/routing/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+// This file contains route calculation utilities that can be used by the
+// survey.
+import moment from 'moment';
+import { RoutingOrTransitMode } from 'chaire-lib-common/lib/config/routingModes';
+import { getPointCoordinates } from 'chaire-lib-common/lib/services/geodata/GeoJSONUtils';
+import { RouteCalculationParameter, RoutingResultByMode } from './types';
+import projectConfig from '../../config/projectConfig';
+import routeWithTransitionApi from './RouteCalculationFromTransition';
+
+/**
+ * Calculate the routes for a list of modes
+ * @param modes The list of modes to calculate the routes for
+ * @param parameters The parameters for the route calculation
+ * @param withGeojson Whether to include the GeoJSON in the response
+ */
+export const calculateRoutesByMode = async function (
```

Comment:
I know we only support Transition for now, but can we have the Transition stuff into a private function that is called by this public one? So that we know where to add a new one easily.

Reply:
It is the case, at least it is the intent. This is the public function that chooses the method to call. Unfortunately, in typescript, we cannot have private file. Technically someone should never import directly RouteCalculationFromTransition, but we cannot force that. Unless we put all the function for all methods in this file... but that sucks

Reply:
Ok, I thought that lot of it was specific for Transition

Reply:
No, parameters needs to be validated for all methods.